src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -924,7 +933,7 @@ class Simplex_tree {
    */
   template<class InputVertexRange = std::initializer_list<Vertex_handle>>
   std::pair<Simplex_handle, bool> insert_simplex_and_subfaces(const InputVertexRange& Nsimplex,
-                    Filtration_value filtration = 0) {
```

Comment:
github interface doesn't let me comment on the documentation part of this method, but it is said:
```txt
   * If the insertion fails (the simplex is already there), the bool is set to false. If the insertion
   * fails and the simplex already in the complex has a filtration value strictly bigger than 'filtration',
   * we assign this simplex with the new value 'filtration', and set the Simplex_handle field of the
   * output pair to the Simplex_handle of the simplex.
  ```
We should explain the behaviour difference when `SimplexTreeOptions::is_multi_parameter = true`